tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1342,6 +1342,13 @@ This can be useful to create Callables to pass around without polluting the clas
     var lambda = func(x): print(x)
     lambda.call(42) # Prints "42"
 
+Unlike Python, lambda functions require an explicit `return`.
+
+::
+
+    var lambda = func(x): return x*x
+    print(lambda.call(2)) # prints "4"
```

Comment:
```suggestion
Note that if you want to return a value from a lambda, an explicit ``return``
is required (you can't omit ``return``)::

    var lambda = func(x): return x ** 2
    print(lambda.call(2)) # Prints `4`.
```

1. Note that lambdas are not required to return a value. I agree with @AThousandShips that the comparison with Python is unnecessary here, since Python uses a special syntax for lambdas, completely different from GDScript.
2. I would also move this paragraph below, after the note about named lambdas.
3. See also godotengine/godot-proposals#5925.